ui/v2.5/src/components/MainNavbar.tsx

```diff
@@ -239,18 +265,20 @@ export const MainNavbar: React.FC = () => {
         </Navbar.Brand>
         <Navbar.Toggle className="order-0" />
         <Navbar.Collapse className="order-3 order-md-1">
-          <Nav className="mr-md-auto">
-            {menuItems.map((i) => (
-              <Nav.Link eventKey={i.href} as="div" key={i.href}>
-                <LinkContainer activeClassName="active" exact to={i.href}>
-                  <Button className="minimal w-100">
-                    <Icon icon={i.icon} />
-                    <span>{intl.formatMessage(i.message)}</span>
-                  </Button>
-                </LinkContainer>
-              </Nav.Link>
-            ))}
-          </Nav>
+          <Fade in={!loading}>
```

Comment:
This `Fade` was added to avoid a noticeable "pop-in" when rendering the navbar for the first time. This is caused by the fact that we need to fetch the navbar items from the config before we can render the items.